include/pybind11/detail/type_caster_base.h

```diff
@@ -241,7 +241,8 @@ struct value_and_holder {
                    ? inst->simple_holder_constructed
                    : (inst->nonsimple.status[index] & instance::status_holder_constructed) != 0u;
     }
-    void set_holder_constructed(bool v = true) const {
+    //NOLINTNEXTLINE(readability-make-member-function-const)
```

Comment:
I just looked, everywhere else we have a space between `//` and `NOLINTNEXTLINE` (which looks much nicer IMM).